prowler/compliance/aws/aws_account_security_onboarding_aws.json

```diff
@@ -0,0 +1,445 @@
+{
+    "Framework": "AWS-Account-Security-Onboarding",
+    "Version": "",
+    "Provider": "AWS",
+    "Description": "Checklist when onboarding new AWS Accounts to existing AWS Organization.",
+    "Requirements": [
+      {
+        "Id": "deploy-an-account-from-predefined-IaC-template",
+        "Name": "Deploy an account from predefined IaC template",
+        "Description": "Deploy an account from predefined IaC template",
+        "Attributes": [
+          {
+            "Section": "Predefine-IAM-Roles",
+            "Name": "Predefine IAM Roles"
+          },
+          {
+            "Section": "Enabled-security-services",
+            "Name": "Enabled security services"
+          },
+          {
+            "Section": "IDC-integration-SSO-configuration",
+            "Name": "IDC integratio and SSO configuration for AWS Account"
+          },
+          {
+            "Section": "Organizarion-invitation",
+            "Name": "Organizarion invitation"
+          },
+          {
+            "Section": "Billing-emergency-and-security-contacts",
+            "Name": "Billing, emergency, and security contacts"
+          },
+          {
+            "Section": "Root-user-distribution-email-MFA",
+            "Name": "Root user have distribute email + MFA"
+          },
+          {
+            "Section": "S3-Block-Public-Access",
+            "Name": "S3 Block Public Access"
+          },
+          {
+            "Section": "Disable-AMI-public-sharing",
+            "Name": "Disable AMI public sharing"
+          }
+        ],
+        "Checks": []
```

Comment:
There are several checks that apply to these sections, check it again please.

Reply:
Done!

Reply:
Done!